app/lib/search/updater.dart

```diff
@@ -55,7 +55,8 @@ class BatchIndexUpdater implements TaskRunner {
         final int count = _snapshot.documents.length;
         _logger
             .info('Got $count packages from snapshot at ${_snapshot.updated}');
-        await packageIndex.addPackages(_snapshot.documents.values);
+        await packageIndex.addPackages(
+            _snapshot.documents.values.where((pd) => pd.isDeprecated != true));
```

Comment:
*Maybe* add a little note here, so some helpful person doesn't accidentally change it to `!pd.isDeprecated`.